### DISCUSSION TITLE
Option to show child comments count without opening comment toolbar.

### DISCUSSION THREAD

**Discussion by lewdpatriot** [Docile-Alligator/Infinity-For-Reddit#869] [Ideas]
Currently it has to be like this
![IMG_20220801_122923](https://user-images.githubusercontent.com/63286528/182090796-033f8ef3-c9d6-45d2-b89f-20fc6b24993e.jpg)
Which feels very cluttered.
I want something like this
![IMG_20220801_121301](https://user-images.githubusercontent.com/63286528/182090955-906f9eeb-d438-4cfc-a22c-c2b45b1520af.jpg)
Btw I have enabled this option but it doesn't work. Only shows upvote count when toolbar is hidden. 
![Screenshot_2022-08-01-12-22-20-86_2656fe66d4cc08342e93fbdd9fd1d373](https://user-images.githubusercontent.com/63286528/182091071-cb9f4af5-a7c4-48a3-ac61-b23597f62081.jpg)

**Comment by Wladefant** [comment]
Please add the last issue to its own issue post

And I think the requested feature is very contraintuitive, because the child comments are an answer to the parent comment and should be under the parent.

**Reply by lewdpatriot** [reply]
Ok, I didn't know if it's a bug or if the feature isn't implemented yet. So I posted it here
